README.md

```diff
@@ -1,2 +1,22 @@
-# visualize-algorithms
-:fire: Visualize Algorithms
+<h1 align="center">Visualize Algorithms</h1>
+ <br>
+ <img src="https://github.com/SarahLizDettloff/visualize-algorithms/blob/master/screenshot.png?raw=true" alt="Visualize Algorithms"> </br>
```

Comment:
No need of screenshot !
Remove Screenshot image also.

Reply:
Image has been removed. Please see newest commit.